src/Entities/PropertyItems/RichDate.php

```diff
@@ -70,6 +68,14 @@ public function getEnd(): ?DateTime
         return $this->end;
     }
 
+    /**
+     * @return bool
+     */
+    public function getHasTime(): bool
```

Comment:
Should we call this method "hasTime" instead?

Reply:
yes! done ✅